webpack/assets/javascripts/foreman_template_kind.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable jquery/no-val */
+
+import $ from 'jquery';
+
+import store from './react_app/redux';
+import * as EditorActions from './react_app/components/Editor/EditorActions';
+
+export const initTemplateKind = () => {
+  const value = $('#provisioning_template_template_kind_id').val();
```

Comment:
We want to get rid of jquery in the future, so if we do not need to, it's better not to use it, these are quite simple without jquery:

```suggestion
  const value = document.getElementById('provisioning_template_template_kind_id').value;
```